<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cadastro de candidatos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Pretensão Salarial:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Experiência:</string>
    </property>
    <property name="buddy">
     <cstring>até 400 caracteres</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Última Escolaridade:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nomeCandidato">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="pretensaoSalarial">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="experiencia">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>371</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ultimaEscolaridade">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoCadastrarCandidato">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>460</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(135, 255, 245);
color: rgb(3, 28, 255);</string>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoVoltarCandidato">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>460</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(135, 255, 245);
color: rgb(3, 28, 255);</string>
    </property>
    <property name="text">
     <string>voltar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>(400 caracteres)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>utilize ponto ao invés de vírgula.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string> Candidato confirmar email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailCandidato">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
